src/osd/ReplicatedPG.cc

```diff
@@ -9161,12 +9161,12 @@ ObjectContextRef ReplicatedPG::get_object_context(const hobject_t& soid,
       pg_log.get_log().objects.find(soid)->second->op ==
       pg_log_entry_t::LOST_REVERT));
   ObjectContextRef obc = object_contexts.lookup(soid);
-  osd->logger->inc(l_osd_object_ctx_cache_total);
   if (obc) {
     osd->logger->inc(l_osd_object_ctx_cache_hit);
     dout(10) << __func__ << ": found obc in cache: " << obc
 	     << dendl;
   } else {
+    osd->logger->inc(l_osd_object_ctx_cache_total);
```

Comment:
okay, that make sense,  in my opinion, the "object_ctx_cache_total" should record the total cached object_ctx, but if it refers to how many lookups for object_ctx according to comments